Tests/FoundationTests/NSAttributedStringExtensionsTests.swift

```diff
@@ -70,6 +70,23 @@ final class NSAttributedStringExtensionsTests: XCTestCase {
     }
 
     // MARK: - Methods
+    func testApplying() {
+        #if canImport(AppKit) || canImport(UIKit)
+        let string = NSAttributedString(string: "Applying")
+        let out = string.applying(attributes: [
+            .strikethroughStyle: NSNumber(value: NSUnderlineStyle.single.rawValue as Int),
+            .foregroundColor: UIColor.red
```

Comment:
This should fix your compilation errors
```suggestion
            .foregroundColor: Color.red
```